pkgs/applications/science/math/cplex/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv, makeWrapper, openjdk, gtk2, xorg, glibcLocales, releasePath ? null }:
```

Comment:
Alternate suggestion I think I'd prefer: Use the `config` attribute here instead, and use `config.cplex.releasePath` to get the path. Then users using nix-env will have it easy to make it work, it just needs a `{ cplex.releasePath = ...; }` in their `~/.config/nixpkgs/config.nix`. This could then also be documented in the error message: "Set `cplex.releasePath = /path/to/download;` in your ~/.config/nixpkgs/config.nix for `nix-*` commands, or `config.cplex.releasePath = /path/to/download;` in your `configuration.nix` for NixOS ..."

Reply:
No need to apologize, I appreciate the feedback as I am quite a newbie here, and is good to have some mentoring. I like the config idea, it should also be ported to renoise to imho (I inspired myself from the renoise build for this one).

Reply:
Here it is. I hope I did it the right way...